src/Tribe/Commerce/PayPal/Endpoints/Template_Interface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Class Tribe__Tickets__Commerce__PayPal__Endpoints__Template_Interface
+ *
+ * @since TBD
+ */
+interface Tribe__Tickets__Commerce__PayPal__Endpoints__Template_Interface {
+
+	/**
+	 * Registers the resources this template will need to correctly render.
+	 */
+	public function register_resources();
+
+	/**
+	 * Builds and returns the date needed by this template.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $template_data
+	 */
+	public function get_template_data( array $template_data = array() );
+
+	/**
+	 * Enqueues the resources needed by this template to correctly render.
+	 *
+	 * @since TBD
+	 */
+	public function
+	enqueue_resources();
```

Comment:
Oooo! A new standard, but if we adopt this it should be:

```php
/**
 * Enqueues
 * the
 * resources
 * needed
 * by
 * this
 * template
 * to
 * correctly
 * render.
 *
 * @since TBD
 */
public
function
enqueue_resources();
```

Reply:
I liked the idea for 5 seconds. Now I do not like it anymore.